src/services/game.js

```diff
@@ -0,0 +1,21 @@
+import config from '../../config';
+
+/*@ngInject*/
+function GameService($resource) {
```

Comment:
Angular can make accessing a REST API quite awesome. 
ES6 syntax makes it even better.
The parameter lists are kinda optional... It prevents sending data you didn't intend.
Also it's **cool to see the auto-complete/suggestions come through in VS Code**.